tap-agent/src/agent/sender_account.rs

```diff
@@ -1196,8 +1212,9 @@ pub mod tests {
         ActorRef<SenderAllocationMessage>,
         JoinHandle<()>,
     ) {
-        let (mock_sender_allocation, triggered_rav_request, next_unaggregated_fees) =
+        let (mock_sender_allocation, rav_request_result) =
             MockSenderAllocation::new_with_triggered_rav_request();
+        let (triggered_rav_request, next_unaggregated_fees) = rav_request_result.unwrap();
```

Comment:
You are unwrapping the result from `new_with_triggered_rav_request`. Isn't removing the result better?